Don't define CURSES_WIDE, so it doesn't try to use unsupported
types.

Index: c3270/screen.c
--- c3270/screen.c.orig
+++ c3270/screen.c
@@ -66,6 +66,8 @@
 
 #include "cscreen.h"
 
+#undef CURSES_WIDE
+
 /*
  * The usual x3270 COLS variable (current number of columns in the 3270
  * display) is called cCOLS in c3270, to avoid a conflict with curses COLS (the
